elastic/kibana
# [Metrics UI] Remove UUID from Alert Instance IDs

## Summary

Fixes #69623

Metric threshold and inventory alerts will now generate alert instance IDs using only `group` instead of `uuid` and `group`. This was creating problems with throttling and stability for users with multiple Kibana instances.

It turns out we didn't need a unique `uuid` to differentiate alert instances. Multiple alerts can create an instance named `*` without stepping on each other; what matters is that the ID is unique **within** an alert itself. 

This PR also updates the Alerts plugin readme to clarify this.

To test:

- Create a metric alert that with a throttle period (make sure it will definitely fire)
- Go to Alert Management and wait for it to generate instances
- Restart Kibana (try just adding `//` to a file to force a server restart)
- Refresh the Alert Management page
- Ensure that the same instances are there, and that the Duration field hasn't reset

### Checklist
